src/main/java/ome/services/fulltext/FullTextBridge.java

```diff
@@ -55,9 +55,9 @@
  *
  * @author Josh Moore, josh at glencoesoftware.com
  * @since 3.0-Beta3
- * @see <a href="https://trac.openmicroscopy.org.uk/ome/FileParsers">Parsers</a
+ * @see <a href="https://omero.readthedocs.io/en/stable/developers/Search/FileParsers.html">Parsers</a
  *      href>
```

Comment:
minor ``</a href>`` => ``</a>``

Reply:
Addressed in 8a5707bee91dbf53c3cd6ab27036032ab101bbaa